web/styles/right_sidebar.css

```diff
@@ -194,7 +194,7 @@ $user_status_emoji_width: 24px;
 }
 
 .user_sidebar_entry .selectable_sidebar_block {
-    width: calc(100% - 16px);
+    width: calc(100% - 26px);
```

Comment:
How is 26px calculated?

Reply:
It's a bit arbitrary, but not more arbitrary than it was before. I was just trying to get them to not overlap where they did before and didn't after I made the change

before

![before](https://github.com/zulip/zulip/assets/5634097/925e6b4f-dc9a-4eae-84cf-38961fc5b3c2) 

after

![after](https://github.com/zulip/zulip/assets/5634097/f446f04d-20b9-4bd6-89ad-e6a14e3999ee)

Reply:
I'm open to doing something more specific or intentional if there's something you'd like to do here.